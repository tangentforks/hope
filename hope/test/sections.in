(-)(3, 1);
(3-) 1;
(-1) 3;
(-);
(1-);
(-1);

dec minus: num -> num;
--- minus <= (0-);

display;
