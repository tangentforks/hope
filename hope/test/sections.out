>> 2 : num
>> 2 : num
>> 2 : num
>> (-) : num # num -> num
>> lambda x' => 1 - x' : num -> num
>> lambda x' => x' - 1 : num -> num

dec minus : num -> num;
--- minus <= (0 -);
